.github/workflows/ci.yml

```diff
@@ -17,272 +17,275 @@
-      - name: "Static checks"
-        if: success()
+          python-version: '3.x'
+      - name: Cache pre-commit env
+        uses: actions/cache@v1
         env:
-          PYTHON_VERSION: 3.6
-        run: ./scripts/ci/ci_run_all_static_checks.sh
+          cache-name: cache-pre-commit
+        with:
+          path: ~/.cache/pre-commit
+          key: ${{ env.cache-name }}-${{ github.job }}-${{ hashFiles('.pre-commit-config.yaml') }}
+      - name: "Free space"
+        run: ./scripts/ci/ci_free_space_on_ci.sh
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Pylint checks for tests"
+        run: |
+          python -m pip install pre-commit \
+              --constraint requirements/requirements-python${PYTHON_MAJOR_MINOR_VERSION}.txt
+          ./scripts/ci/ci_run_static_checks.sh pylint-tests mypy
 
-  statics-tests:
-    name: Pylint for tests
+  static-checks-2:
+    name: "Checks: All other"
     runs-on: ubuntu-latest
     env:
-      PYTHON_VERSION: 3.6
-      AIRFLOW_MOUNT_SOURCE_DIR_FOR_STATIC_CHECKS: "true"
-      TRAVIS_JOB_NAME: "Static"
+      SKIP: pylint-tests,mypy
+      MOUNT_SOURCE_DIR_FOR_STATIC_CHECKS: "true"
+      CI_JOB_TYPE: "Static checks"
     steps:
       - uses: actions/checkout@master
       - uses: actions/setup-python@v1
         with:
-          python-version: ${{ env.PYTHON_VERSION }}
-      - name: "Before install"
-        run: ./scripts/ci/ci_before_install.sh
-      - name: "Static checks tests"
-        if: success()
+          python-version: '3.x'
+      - name: Cache pre-commit env
+        uses: actions/cache@v1
         env:
-          PYTHON_VERSION: 3.6
-        run: ./scripts/ci/ci_run_static_checks_pylint_tests.sh
+          cache-name: cache-pre-commit
+        with:
+          path: ~/.cache/pre-commit
+          key: ${{ env.cache-name }}-${{ github.job }}-${{ hashFiles('.pre-commit-config.yaml') }}
+      - name: "Free space"
+        run: ./scripts/ci/ci_free_space_on_ci.sh
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Static checks"
+        run: |
+          python -m pip install pre-commit \
+              --constraint requirements/requirements-python${PYTHON_MAJOR_MINOR_VERSION}.txt
+          ./scripts/ci/ci_run_static_checks.sh
 
   docs:
     name: Build docs
     runs-on: ubuntu-latest
     env:
-      TRAVIS_JOB_NAME: "Build documentation"
-      PYTHON_VERSION: 3.6
+      CI_JOB_TYPE: "Documentation"
     steps:
       - uses: actions/checkout@master
-      - name: "Build documentation"
+      - name: "Build CI image ${{ matrix.python-version }}"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Build docs"
         run: ./scripts/ci/ci_docs.sh
 
-  tests-p36-postgres-integrations:
-    name: "Tests [Postgres9.6][Py3.6][integrations]"
+  build-prod-image:
+    name: "Build prod image"
     runs-on: ubuntu-latest
-    needs: [statics, statics-tests]
```

Comment:
Is there any particular reason for skipping this? (Or is it just for faster feedback during development?)

Reply:
It's still WIP. Unfortunately the tests on GA are super-unstable and I am trying to find a way to stabilise them.